client/src/components/Connections/scenes/Results/components/InfoLoader/InfoLoader.scss

```diff
@@ -0,0 +1,32 @@
+.resultLoading {
+  background-color: #ffffff;
+  width: 400px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 30px;
```

Comment:
it's also good to use rem on margins/paddings where it makes sense, as it scales better with the rest of the page, but I don't know the exact good practices here so keeping in px might be ok

Reply:
I think the whole design and UX has to be reworked, so I don't want to dig into too much polishing.